### ISSUE TITLE
Upload::get_errors() returns files with UPLOAD_ERR_OK

### ISSUE THREAD

**Issue by zoe-edwards** [fuel/core#458]
```
Upload::process();

if (Upload::is_valid())
{
    Upload::save();
}

print_r(Upload::get_errors());
```

returns

```
Array
(
    [0] => Array
        (
            [name] => my-image.png
            [type] => image/png
            [error] => 109
            [size] => 290412
            [field] => userfile
            [key] => 
            [file] => /Applications/MAMP/tmp/php/phpx2ynhp
            [extension] => png
            [filename] => my-image
            [mimetype] => image/png
            [message] => The file uploaded with success
            [saved_to] => uploads/
            [saved_as] => my-image.png
        )

)
```

The Docs state:

> The get_files method returns a multi-dimensional array with all files uploaded that have an error status != Upload::UPLOAD_ERR_OK.

**Comment by WanWizard** [comment]
get_errors() returns the errors, which is correct, as per your print_r(), the error code is 109, which is != UPLOAD_ERR_OK.

The error message is incorrect though, due to a bug in save(), which has been fixed in 1.1/develop.

**Comment by zoe-edwards** [comment]
Had the permissions of the folder set wrong! Cheers Harro.
